webapp-django/crashstats/settings/base.py

```diff
@@ -631,7 +633,7 @@ def filter(self, record):
             ),
         },
     },
-    "telemetrydata": {"bucket_name": config("TELEMETRY_BUCKET_NAME", None)},
+    "telemetrydata": {"bucket_name": config("destination.telemetry.bucket_name", None)},
```

Comment:
`destination.telemetry.bucket_name` is what the processor uses. Given that we're using Socorro configman names in other places, I figured we should change this one, too. Fewer places to have to change things in the future.